updater/cmd/updater/main.go

```diff
@@ -32,10 +34,10 @@ import (
 	"go.uber.org/zap"
 )
 
-// Unimplemented
 func main() {
-	var showVersion = pflag.BoolP("version", "v", false, "Prints the version of the collector and exits, if specified.")
+	var showVersion = pflag.BoolP("version", "v", false, "Prints the version of the updater and exits, if specified.")
 	var tmpDir = pflag.String("tmpdir", "", "Temporary directory for artifacts. Parent of the 'rollback' directory.")
+	var logLevelStr = pflag.String("loglevel", "debug", "Level with which to log")
```

Comment:
If the default is `debug` is there any real reason to specify this? Users can't hook into this and it's not easy to inject a value for ourselves. I think just having `debug` logging is fine.

Reply:
I was thinking that maybe the collector could at some point dynamically change this based on the collector log level, but you're right, we should just hard code for now.